packages/frontend/src/components/staking/components/Staking.js

```diff
@@ -105,6 +106,24 @@ export default function Staking({
                     />
                 </>
                 : null}
+            {!loading && collectedFarmData.map((tokenData, i) => {
+                return (
```

Comment:
I think we should move the filtering for zero balance rewards here for reusability purposes and in keeping with the other `BalanceBox` logic on this component

Reply:
Done